### ISSUE TITLE
Couldn't load font

### ISSUE THREAD

**Issue by OfficialJens** [AKORA-Studios/DiscordWelcomeCard#10]
**Error Message:**
Pango-WARNING **: 12:21:17.238: couldn't load font "segoe-ui-emoji Not-Rotated 30px", falling back to "Sans Not-Rotated 30px", expect ugly output.

**Code**
Steps to reproduce the behavior / code example
```
    const image = await drawCard({
        theme: 'dark',
        text: {
            title: 'Welcome to Allstar,',
            text: guildMember.user.tag,
            subtitle: 'please read the Rules',
            color: `#FFF`,
        },
        avatar: {
            image: guildMember.displayAvatarURL({ format: 'png' }),
            outlineWidth: 5,
        },
        background: 'https://i.imgur.com/ea9PB3H.png',
        blur: 1,
        border: true,
        rounded: true
    });
```

**Additional context**
Discord JS 13, TypeScript.

**Comment by A-K-O-R-A** [comment]
Can you provide the System/OS you are running this code on?

Installing Cairo/Pango might help if you are using linux
`sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev`

**Comment by OfficialJens** [comment]
Dev Host: Windows 11
Final OS: Ubuntu

**Comment by A-K-O-R-A** [comment]
Maybe try using the prerelease version [4.1.0-dev.0](https://www.npmjs.com/package/discord-welcome-card/v/4.1.0-dev.0) of the package, it comes with its own font and should fix your problem.
`npm i npm i discord-welcome-card@4.1.0-dev.0`

**Comment by A-K-O-R-A** [comment]
Another option is to fall back to old fonts by installing `ttf-ancient-fonts`, for ubuntu that would be `sudo apt-get install ttf-ancient-fonts`.
For me, this made the difference pretty minor as seen in this picture, on the left is the image generated on WIndows while the image on the right is generated on a Ubuntu server running Node v16 
![grafik](https://user-images.githubusercontent.com/65976562/155603282-a9923bbd-c380-479a-8031-a9edb71b1cee.png)

**Comment by A-K-O-R-A** [comment]
I just released [v4.2.0](https://www.npmjs.com/package/discord-welcome-card/v/4.2.0) with 2 Fonts included, one for fallback, with this the results on my Linux machine are almost identical to the results on my windows machine.
![grafik](https://user-images.githubusercontent.com/65976562/155605065-d8c07089-7ec2-44e2-bd58-9611c14327ce.png)
I would be thankful if you could install the newest version and test your code again, to see if this solves the issue not just on my machine but generally.

**Comment by DragonCat4012** [comment]
@OfficialJens 

due to inactivity we will close this issue in a few days
